tests/integration/targets/win_hotfix/aliases

```diff
@@ -0,0 +1,2 @@
+shippable/windows/group4
```

Comment:
While the test is unstable so won't run we should make sure this is in a valid group

```suggestion
shippable/windows/group2
```